lib/routes/user/marketplace/marketplace.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:breez/theme_data.dart' as theme;
+import 'package:breez/widgets/back_button.dart' as backBtn;
+
+class MarketplacePage extends StatefulWidget {
+  final String _title = "Marketplace";
+
+  @override
+  State<StatefulWidget> createState() {
+    return new MarketplacePageState();
+  }
+}
+
+class MarketplacePageState extends State<MarketplacePage> {
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+        appBar: new AppBar(
+          leading: backBtn.BackButton(),
+          automaticallyImplyLeading: false,
+          iconTheme: theme.appBarIconTheme,
+          textTheme: theme.appBarTextTheme,
+          backgroundColor: Color.fromRGBO(5, 93, 235, 1.0),
+          title: new Text(
+            widget._title,
+            style: theme.appBarTextStyle,
+          ),
+          elevation: 0.0,
+        ),
+        body: ListView(
+          children: <Widget>[
```

Comment:
It will be nice to have the ListItem (Bitrefill in this case) rendered in its own widget, we can call it "Seller".
Later we will probably have a "MarketPlaceBloc" where all the sellers will be fetched from there. Right not we don't have to implement the bloc but as soon as we have a new seller we should do it.